docs/machine-learning/index.yml

```diff
@@ -1,69 +1,107 @@
-### YamlMime:YamlDocument
-documentType: LandingData
+### YamlMime:Landing
+
 title: ML.NET Documentation
+summary: Learn how to use ML.NET, an open-source framework, to build machine learning solutions and integrate them into applications. Tutorials, code examples, and more.
```

Comment:
Is there a bit of text missing at the end of this:

`summary: Learn how to use ML.NET, an open-source framework, to build machine learning solutions and integrate them into applications. Tutorials, code examples, and more.`

--> add "show you how"?

Reply:
Length requirements are strict, so I opted to use short hand. Otherwise, I'd have to cut something.

Reply:
Length is at 160, and the current version is exactly that. Option B (158 chars): 
Learn how to use open-source ML.NET to build machine learning solutions and integrate them into applications. Tutorials, code examples, and more show you how.

Preference?

Reply:
@natke - going with Option B per conversation. :)